app/views/tools/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<% title(t('tool.title')) %>
+<div class="col-md-8 col-md-offset-2 col-sm-12">
+  <div class="headline">
+    <h1><%= @tool.title %></h1>
```

Comment:
Do you want the page title to be different here? Otherwise it could be:
`<h1><%= title(@tool.title) %></h1>`